cells/addf/definition.json

```diff
@@ -0,0 +1,53 @@
+{
+    "description": "Full Adder, 2X drive strength",
```

Comment:
These shouldn't have the `, 2X drive strength` as there are multiple drive strength versions of the Full adder under the `addf` cell.

Reply:
Done